datastore/noxfile.py

```diff
@@ -3,7 +3,7 @@
 import nox
 
 
-LOCAL_DEPS = ('../auth/', )
+LOCAL_DEPS = ('../auth/', '../storage/', )
```

Comment:
This is probably going to make independent versioning of `storage` and `datastore` difficult, for example when we've updated one but not the other to use incompatible dependencies. Let's leave this off for now, disable the new test by default, and mark the skip message with "requires gcloud-aio-storage" so we can run it adhoc for now.

Reply:
k, well not sure the independent versioning is important since this dependency is solely for unit tests. If in the future datastore ever does depend on storage then it can be removed. But if you want it removed we can, will just make running the export test more complicated.